release notes/v0.24.0.md

```diff
@@ -66,7 +66,7 @@ For a list of all current error codes, see the docs page [here](https://docs.k6.
 - Most of the HTTP request code has been refactored out of the `js` packages and is now independent from the goja JS runtime. This was done mostly so we can implement the error codes feature (#907), but will allow us more flexibility in the future. (#928)
 - As a preparation for the upcoming big refactoring of how VUs are scheduled in k6, including the arrival-rate based execution, we've added the future `execution` configuration framework. It currently doesn't do anything besides warn users that use execution option combinations that won't be supported in future k6 versions. See the Breaking Changes section in these release notes for more information. (#913)
 - Switched to [golangci-lint](https://github.com/golangci/golangci-lint) via [golangci.com](https://golangci.com/) for code linting in this repo. The gometalinter check in CircleCI is still enabled as well, but it will be removed in the following few weeks. (#943)
-- Switched to Go 1.12.1 for building and testing k6, removed official support with 1.10. (#944)
+- Switched to Go 1.12.1 for building and testing k6, removed official support with 1.10. (#944 and #966)
```

Comment:
remove official support *for* 1.10 
or
, and dropped 1.10 as build and test target